.github/workflows/create-caches.yml

```diff
@@ -29,42 +46,62 @@ jobs:
         with:
           path: models/ldm/stable-diffusion-v1/model.ckpt
           key: ${{ env.cache-name }}
-          restore-keys: |
-            ${{ env.cache-name }}
+          restore-keys: ${{ env.cache-name }}
+
+      - name: Use Cached Stable Diffusion v1.4 Model
```

Comment:
During my (unfortunately very lengthy and time-consuming 🙄) testing when *I* was trying to get Windows working, without success, I noticed that on average it's *much* faster to download the model every time ***because*** when you stash something in the lookaside cache to reuse, it gets zipped.

So instead of a straightforward download from Hugging Face's servers, there's a much more complicated download/unzip/move/not sure what else which *usually* takes longer (and yeah, this is across all three platforms).

Reply:
To be honest - I didn't find out how I need to create the secret with a token in a way that it is valid as long as the token - I cheated on my side with this secret (while still obtaining it from the huggingface site), but this was about 50% the speed I get from downloading it out of the cache (and the cache would still be valid after the secret has already lost it's authorization 🙈)

Reply:
Right, I forgot about that - I had to set up *my* token for it to work properly. The main repo should never have this issue, and if it does, it's something which we'd need to fix immediately

Reply:
So should I disable the caching and always download, r u sure? imho this can be rly helpfull when there are many runs and saves bandwith of the backbone for people watching cat videos on YT ;P

Reply:
> imho this can be rly helpfull when there are many runs

That's what I'm saying - **it isn't**. I was absolutely stunned by this at first, so I went down a total rabbithole of stopping work on getting Windows working just to focus on the model download.

ON AVERAGE, downloading the model is faster (by a lot) than caching it.

OTOH, "saves bandwith of the backbone" is another reason why we want to limit the number of times we run the suite.

Reply:
So for me the download from huggingface usually had betwen 50-100MB/s while GH-Cache had about 150MB/s, which is also handled internally and not over the web (where I am not sure if downloading from huggingface is maybe slower because it needs to be routed externally), which would lead to --> using cache doesn't bother the backend at all ;P